### DISCUSSION TITLE
Installation error：make install error

### DISCUSSION THREAD

**Discussion by humouren0430** [sylabs/singularity#3408] [General]
hufy@ps:~/regression_by_myself/10.22-10.28/dorado-release-v0.8/singularity-ce-4.2.0/builddir$ make
 GO singularity
    [+] GO_TAGS "sylog singularity_engine fakeroot_engine apparmor selinux seccomp"
# github.com/sylabs/singularity/v4/pkg/util/loop
../pkg/util/loop/loop_singularity.go:22:26: undefined: buildcfg.SINGULARITY_CONF_FILE
# github.com/sylabs/singularity/v4/internal/pkg/util/starter
../internal/pkg/util/starter/starter.go:60:36: undefined: buildcfg.LIBEXECDIR
../internal/pkg/util/starter/starter.go:63:35: undefined: buildcfg.LIBEXECDIR
../internal/pkg/util/starter/starter.go:160:35: undefined: buildcfg.ENGINE_CONFIG_ENV
../internal/pkg/util/starter/starter.go:161:35: undefined: buildcfg.ENGINE_CONFIG_CHUNK_ENV
../internal/pkg/util/starter/starter.go:183:21: undefined: buildcfg.MAX_ENGINE_CONFIG_CHUNK
../internal/pkg/util/starter/starter.go:184:29: undefined: buildcfg.MAX_CHUNK_SIZE
../internal/pkg/util/starter/starter.go:195:19: undefined: buildcfg.MAX_ENGINE_CONFIG_CHUNK
../internal/pkg/util/starter/starter.go:196:73: undefined: buildcfg.MAX_ENGINE_CONFIG_SIZE
../internal/pkg/util/starter/starter.go:221:34: undefined: buildcfg.LIBEXECDIR
# github.com/sylabs/singularity/v4/internal/pkg/util/bin
../internal/pkg/util/bin/bin_singularity.go:42:45: undefined: buildcfg.SINGULARITY_CONF_FILE
../internal/pkg/util/bin/bin_singularity.go:74:45: undefined: buildcfg.SINGULARITY_CONF_FILE
../internal/pkg/util/bin/bin_singularity.go:103:14: undefined: buildcfg.CONMON_LIBEXEC
../internal/pkg/util/bin/bin_singularity.go:104:33: undefined: buildcfg.LIBEXECDIR
../internal/pkg/util/bin/bin_singularity.go:113:14: undefined: buildcfg.SQUASHFUSE_LIBEXEC
../internal/pkg/util/bin/bin_singularity.go:114:33: undefined: buildcfg.LIBEXECDIR
../internal/pkg/util/bin/bin_singularity.go:127:32: undefined: buildcfg.LIBEXECDIR
# github.com/sylabs/singularity/v4/internal/pkg/remote
../internal/pkg/remote/remote.go:45:47: undefined: buildcfg.SYSCONFDIR
# github.com/sylabs/singularity/v4/internal/pkg/signature
../internal/pkg/signature/verify.go:237:38: undefined: buildcfg.SINGULARITY_CONFDIR
make: *** [Makefile:181: singularity] Error 1

**Comment by humouren0430** [comment]
i don't know why,i have struggled in this step for five days

**Comment by dtrudg** [comment]
You would need to provide more information about what you have done prior to the `make` step. E.g. 

What is the operating system, Go version, etc.

How did you get the code - .tar.gz download, git clone etc.

How did you run mconfig etc.

**Comment by humouren0430** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
